src/webrtc/callFeed.ts

```diff
@@ -184,21 +184,18 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
     }
 
     /**
-     * Set feed's internal audio mute state
-     * @param muted is the feed's audio muted?
-     */
-    public setAudioMuted(muted: boolean): void {
-        this.audioMuted = muted;
-        this.speakingVolumeSamples.fill(-Infinity);
-        this.emit(CallFeedEvent.MuteStateChanged, this.audioMuted, this.videoMuted);
-    }
-
-    /**
-     * Set feed's internal video mute state
+     * Set one or both of feed's internal audio and video video mute state
+     * Either value may be null to leave it as-is
      * @param muted is the feed's video muted?
      */
-    public setVideoMuted(muted: boolean): void {
-        this.videoMuted = muted;
+    public setAudioVideoMuted(audioMuted: boolean, videoMuted: boolean): void {
```

Comment:
Marked this as a breaking change, any chance you fancy keeping the old ones around for compatibility but just calling the new one under the hood?

Reply:
This is just internal js-sdk API, so shouldn't need to be a breaking change (although of course it's a public method on an object that the API consumer gets given, so they could call it, but it wouldn't really do anything useful).